build_scripts/npm_macos/package.json

```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "appdmg": "^0.6.4",
```

Comment:
Is this being used?

Reply:
Not entirely sure, but it was in one of the previous package.json files for mac, and was in both lock files.

Reply:
Ahh ok, it's a dependency of electron-installer-dmg

Reply:
Looks to be used by electron-installer-dmg, so it probably doesn't need to be specifically called out like it is, but used either way

```
├─┬ electron-installer-dmg@3.0.0
│ ├── appdmg@0.6.4 deduped
```